src/System.Buffers.Experimental/System/Buffers/Text/BufferWriter_strings.cs

```diff
@@ -0,0 +1,200 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Utf8;
+
+namespace System.Buffers.Text
+{
+    public ref partial struct BufferWriter
+    {
+        #region String
+        public bool TryWrite(string text)
+        {
+            if (!TryWrite(text, out var written)) return false;
```

Comment:
nit: specify the type instead of using `var`

Reply:
will do